core/tests/test_utils.py

```diff
@@ -3755,9 +3778,11 @@ class CallCounter(FunctionWrapper):
     increased when the function raises an exception.
     """
 
-    def __init__(self, f):
+    def __init__(self, f=lambda *_, **__: None):
```

Comment:
I don't understand this -- where do we use the default args? What is the point of defining a CallCounter if there's no function to track calls on?

Reply:
I reverted the changes since I'm using `swap_with_call_counter` instead.